mnist/HOWTO.greene.md

```diff
@@ -24,6 +24,7 @@ Inside the python venv, with the right version python3 and pip3, you can install
 pip3 install torch
 pip3 install torchsummary
 pip3 install -U numpy
+pip3 install torchsummary
```

Comment:
I do not see torchvision. :)

Reply:
Apologies. Added it now. Thank you, Professor.